pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.poetry]
+name = "mycompany-consent-demo"
+version = "0.1.0"
+description = ""
+
+[tool.poetry.dependencies]
+python = "^3.7"
```

Comment:
Could we update this to require python 3.7 so it's in line with the version required in the backend directory?